### PR TITLE
Npm publish

### PR DESCRIPTION
**Pull request by steveoni**
this pull request handles the publishing of giftless-client to npm

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -58,11 +58,13 @@ export default class Client {
     return url;
   }
 
-  _urlencode = function(obj) {
+  _urlencode(obj) {
```
Comment (cotts) [inline src/index.js:61]: function and methods should be `camelCase`

File: src/index.js [hunk 2]
```diff
@@ -58,11 +58,13 @@ export default class Client {
...
     let str = [];
     for (let key in obj) {
       str.push(encodeURIComponent(p) + "=" + encodeURIComponent(obj[p]));
```
Comment (cotts) [inline src/index.js:64]: where this `p` variable is defined?

**Review by cotts — CHANGES_REQUESTED** [review]
Awaiting review

File: src/index.js (hunk 1 above)
Comment (steveoni) [reply src/index.js:61]: I'd updated this last week, I will push the new changes

File: src/index.js (hunk 2 above)
Comment (steveoni) [reply src/index.js:64]: oh, thanks for this, it should be key, when copying my test code from the browser console

File: src/index.js (hunk 1 above)
Comment (steveoni) [reply src/index.js:61]: this a private method though, is there a need for camelCase

**Review by cotts — APPROVED** [review]
Looks good 👍🏻

**Pull request merged**